src/global-table.ts

```diff
@@ -1,22 +1,103 @@
-import { IResource, RemovalPolicy, Resource, Stack } from 'aws-cdk-lib';
+import { IResource, RemovalPolicy, Resource, Stack, Token } from 'aws-cdk-lib';
 import * as ddb from 'aws-cdk-lib/aws-dynamodb';
+import * as iam from 'aws-cdk-lib/aws-iam';
 import { Construct } from 'constructs';
+import * as perms from './perms';
 
 export interface IGlobalTable extends IResource {
+
+  /**
+   * Adds an IAM policy statement associated with this table to an IAM
+   * principal's policy.
+   * @param grantee The principal (no-op if undefined)
```

Comment:
Remove "no-op if undefined" from each grant method's documentation. It's not possible for the `grantee` to be undefined. It looks like this is there in `Table` because it was left-over from a refactor done a long time ago: https://github.com/aws/aws-cdk/commit/1bb8ca9f5b3e2b82982c52b492d620f4b3ec07aa